azure-pipelines/templates/steps/auth-client/docker-tasks.yml

```diff
@@ -1,22 +1,47 @@
-# File: docker-tasks.yml
+# File: azure-pipelines\templates\docker-instrumented-test.yml
+# Description: Template to run Instrumented test for Supported API levels
+
+parameters:
+- name: azureContainerRegistryImage
+  default: authclient.azurecr.io/samples/dbi-instrumented-api30
+- name: androidProject
+  default: self
+- name: EnvVstsMvnAccount
+  default: ''
 
 steps:
 - task: PostBuildCleanup@3
   condition: always()
-  displayName: 'Post build clean up'
-- checkout: self
+  displayName: Post build clean up
+- checkout: ${{ parameters.androidProject }}
   clean: true
   submodules: recursive
+- script: touch env.list
+  displayName: 'Create Enviroment file'
+- script: echo "${{ parameters.EnvVstsMvnAccount }}=$(mvnAccessToken)" >> env.list
```

Comment:
Are we trying to append to the file in this case?

Reply:
yes